### PR TITLE
Add honeys.be and moimoi.re domains

### PR DESCRIPTION
**Pull request by edtechd**

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -5606,6 +5606,7 @@
   "avchina.us",
   "avenirshoes.ru",
   "avenueway.com",
+  "aver.com",
```
Comment (kevinwbaker) [inline index.json]: This was removed in https://github.com/ivolo/disposable-email-domains/pull/772

**Comment by danhstevens** [comment]
Hi There. These look like free domains - I can't find a disposable email service related to them. Closing for now. Feel free to reopen if you know of a disposable service that is using them

**Comment by edtechd** [comment]
Hello,
The domains are from the https://m.kuku.lu/en.php service

**Comment by danhstevens** [comment]
Thanks! I couldn't find aver.com but I'll get these others added.

**Comment by edtechd** [comment]
aver.com has been added by mistake as I worked on the pull request in the same time when #772 has been applied

**Pull request closed without merging**